<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableForeignKeyPanel</class>
 <widget class="QWidget" name="TableForeignKeyPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="fkTableWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="fkTableLabel">
        <property name="text">
         <string>Foreign table:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fkTableCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="columnsGroup">
     <property name="title">
      <string>Columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="columnsHeader" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="colHdrLocal">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Local column</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="colHdrForeign">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Foreign column</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="columnsArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="columnsScrollContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>378</width>
           <height>81</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="reactionsGroup">
     <property name="title">
      <string>Reactions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="onUpdateCheckBox">
        <property name="text">
         <string>ON UPDATE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="onUpdateCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="onDeleteCheckBox">
        <property name="text">
         <string>ON DELETE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="matchCheckBox">
        <property name="text">
         <string>MATCH</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="onDeleteCombo"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="matchCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="deferredGroup">
     <property name="title">
      <string>Deferred foreign key</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="deferrableCombo"/>
      </item>
      <item>
       <widget class="QComboBox" name="initiallyCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="namedWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="namedCheckBox">
        <property name="text">
         <string>Named constraint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="placeholderText">
         <string>Constraint name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
